docs/pages/versions/v52.0.0/sdk/notifications.mdx

```diff
@@ -578,20 +576,9 @@ await Notifications.scheduleNotificationAsync({
 
 </Collapsible>
 
-### Android push notification payload specification
-
-When sending a push notification, put an object conforming to the following type as `data` of the notification:
+### Push notification payload specification
 
-```ts
-export interface FirebaseData {
-  title?: string;
-  message?: string;
-  subtitle?: string;
-  sound?: boolean | string;
-  vibrate?: boolean | number[];
-  badge?: number;
-}
-```
+Please refer to the `ExpoPushMessage` type in the [`expo-server-sdk`](https://github.com/expo/expo-server-sdk-node/blob/b470314023d8393073ecdbc07927b32d163af790/src/ExpoClient.ts#L346) package.
```

Comment:
```suggestion
See the `ExpoPushMessage` type in the [`expo-server-sdk`](https://github.com/expo/expo-server-sdk-node/blob/b470314023d8393073ecdbc07927b32d163af790/src/ExpoClient.ts#L346) package.
```

We avoid using the "please" word since docs use more of a neutral language to instruct the developer. Let's remove it here.